.circleci/config.yml

```diff
@@ -828,6 +840,22 @@ workflows:
             - GCP2
 
 
+  # in order to trigger this job
+  # 1. go to the CircleCI dashboard
+  # 2. go to your branch of choice
+  # 3. click "Trigger Pipeline"
+  # 4. Add string paramter "sdk_release_tag" and set to a valid git tag/commit
+  sdk-release:
+    when: << pipeline.parameters.sdk_release_tag >>
+    jobs:
+      - publish-libs:
+          context: Audius Client
+          name: publish-libs (<< pipeline.parameters.sdk_release_tag >>)
+          sdk_release_tag: << pipeline.parameters.sdk_release_tag >>
+          filters:
+            branches:
+              only: /(^master$)/
```

Comment:
We could choose to remove this filter, but then we could potentially run a release using any arbitrary branch. Doing that would not display the release job within the master branch: https://app.circleci.com/pipelines/github/AudiusProject/audius-protocol?branch=master

Additionally, it would only allow us to more easily test modifications to the pipeline, but these other changes would still need to be made: 4cf6bb6bd7da53f6c87a60c256059a04ee11d703.

Let's leave this filter in so that we have all the release jobs visible off the master branch.

Reply:
you're right lets keep it master for now. i was thinking of a more specific use case where we do like dev versions of the sdk package but that's not the major flow we want to support